ui/service.go

```diff
@@ -336,6 +468,21 @@ func (gs *uiService) ViewGroup(ctx context.Context, token, id string) ([]byte, e
 	return btpl.Bytes(), nil
 }
 
+func (gs *uiService) Assign(ctx context.Context, token string, groupID, groupType string, memberIDs ...string) ([]byte, error) {
+	if err := gs.sdk.Assign(memberIDs, groupType, groupID, token); err != nil {
+		return []byte{}, err
+	}
+
```

Comment:
Here you left an empty line, but in the function below you did not. Please choose one or another and stay consistent.